_data/team.yml

```
@@ -1,7 +1,7 @@
 - name: Eleonora Belova
   position: Programme Coordinator
   comment: Managing the Programme
-  image: https://media.licdn.com/dms/image/C4E03AQEV0jEE1z0BbA/profile-displayphoto-shrink_800_800/0/1651913271528?e=1678320000&v=beta&t=zc9ZQRqg5_MBq1m_WRsqRyyZsJ5xDyfnGE6bktR6RgE
+  image: assets/images/team/1.jpeg
```

Please use the existing image

```suggestion
  image: assets/images/mentors/1.jpeg
```